nekoyume/Assets/_Scripts/UI/Scroller/NotificationScroll.cs

```diff
@@ -42,7 +43,7 @@ public void Dispose()
         }
 
         [SerializeField]
-        private FancyScrollView.Scroller scroller = null;
+        private UnityEngine.UI.Extensions.Scroller scroller = null;
```

Comment:
```suggestion
        private Scroller scroller = null;
```

Reply:
This is ambiguous for `UnityEngine.UI.Elements` and `UnityEngine.UI.Extensions`. So I put a in front of it.